.github/CONTRIBUTING.md

```diff
@@ -101,30 +101,30 @@ These instructions assume a parent directory `bar` and we will be adding two chi
     $ mkdir bar && cd bar
     # Clone the forked repo locally
     bar $ git clone https://github.com/YOURNAME/botpress.git
+    # Install the dependencies
+    bar $ ./node_modules/.bin/lerna bootstrap
```

Comment:
Then you probably want `lerna run compile` to build all the packages
I have these instructions in the dev-bot's readme

Reply:
@emirotin , fixed those points.